packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -223,11 +289,21 @@ export default class extends Component {
       }))
     }
 
+    const _isPciPassthroughAvailable = isPciPassthroughAvailable(host)
+    const isPciHiddenById = {}
+    if (_isPciPassthroughAvailable) {
+      await Promise.all(
+        Object.keys(this.props.pcis).map(async id => (isPciHiddenById[id] = await isPciHidden(id).catch(console.error)))
+      )
+    }
```

Comment:
If there's an error, we should show that there's an error. You could simply store an error message in the map, for example, and show an error icon with the message in the table. Otherwise, the loading icon will never stop spinning.